### DISCUSSION TITLE
Register two binary Images of different size and resolution

### DISCUSSION THREAD

**Discussion by fvanwick** [SuperElastix/elastix#881] [Q&A]
Hello everyone,

I am currently working on project, where i want to register 2 images of different modality. One image is a MALDI image with a low resolution (5um). The other image is an fluorescence image of the same section but with a much higher resolution (0.4 um). My goal is to find a good registration. 
I already spend weeks to understand Elastix and all its functionality but I am still bemused. 

Hopefully someone had the same struggle I have and can help me fixing my problems.

**Comment by ntatsisk** [comment]
Hi @fvanwick, have you made any progress with this eventually? 

It is actually hard for the community to help without knowing more details for your problem e.g. how the images look like or what you have tried exactly and why it didn't work. In that context, how you validate a "good registration" is also important (do you visualize check the image or do you have some ground truth?).
